tests/components/airnow/test_config_flow.py

```diff
@@ -6,8 +6,11 @@
 
 from homeassistant import config_entries, data_entry_flow
 from homeassistant.components.airnow.const import DOMAIN
+from homeassistant.const import CONF_API_KEY, CONF_LATITUDE, CONF_LONGITUDE, CONF_RADIUS
 from homeassistant.core import HomeAssistant
 
+from tests.common import MockConfigEntry
+
 
 async def test_form(hass: HomeAssistant, config, setup_airnow) -> None:
```

Comment:
These tests should also be updated to reflect the new options data

Reply:
The `test_form` case is the only legacy one that isn't testing error handling, so it seems like the others would not need options data. I can change it as follows to make sure the initial setup splits out `CONF_RADIUS` into options correctly (`user_input` as a fixture having all four `CONF_*` keys), but not sure if I'm missing something on the other tests.

```py
async def test_form(hass: HomeAssistant, config, options, user_input, setup_airnow) -> None:
    """Test we get the form."""
    result = await hass.config_entries.flow.async_init(
        DOMAIN, context={"source": config_entries.SOURCE_USER}
    )
    assert result["type"] == data_entry_flow.FlowResultType.FORM
    assert result["errors"] == {}

    result2 = await hass.config_entries.flow.async_configure(result["flow_id"], user_input)
    assert result2["type"] == data_entry_flow.FlowResultType.CREATE_ENTRY
    assert result2["data"] == config
    assert result2["options"] == options
```

Reply:
It's fine on this one if you just add an addtional assert on the end testing the options data. No need to modify the whole test